bolt-design-system/bolt
# Fix: remove extra padding for block list's last item

## Jira

http://vjira2:8080/browse/BDS-1006

## Summary

Fixed a bug where the list component having unwanted extra spacing on the bottom.

## Details

I changed the conditional CSS (combination of block display and separator) to apply bottom padding only to items that are not the last item.

## How to test

Run the branch locally and then:

1. Create a dummy page
2. On the dummy page, created a div with same padding (2rem) all around, and give it a yellow background color
3. Put a bolt-list inside the div
4. Set display to block, set separator to solid
5. Make sure the spacing is exactly the same all around
